master/public/redirect_github.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html data-bs-theme="dark" lang="en" data-bss-forced-theme="dark"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no"><title>fxfurtrack</title><meta http-equiv="refresh" content="0; url=https://github.com/Blumlaut/fxfurtrack"><link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css"><link rel="stylesheet" href="assets/css/styles.min.css"></head><body><div id="content"><img class="img-fluid" src="assets/img/fxfurtrack.png" width="562" height="90"><p>Redirecting to <strong>GitHub</strong>...</p><div class="loader"></div></div><script src="assets/bootstrap/js/bootstrap.min.js"></script></body></html>
```

Comment:
references to assets must be absolute, otherwise the browser will end up trying to find `/user/blumlaut/assets/bootstrap...`

Reply:
![bootstrap is kek](https://github.com/user-attachments/assets/efae5a5b-0d63-4ac5-86cc-ab25f8e3193f)